Set the Galactocentric frame parameter defaults using a ScienceState (#9346)
astropy/astropy: docs/coordinates/galactocentric.rst

.. _coordinates-galactocentric:

**************************************************
Description of the Galactocentric Coordinate Frame
**************************************************

While many other frames implemented in `astropy.coordinates` are standardized in
some way (e.g., defined by the IAU), there is no standard Milky Way

> Galactocentric reference frame. The `~astropy.coordinates.Galactocentric` frame

Reviewer: I think this is a good place to defince the term for those who stumble upon it.
```suggestion
reference frame with the center of the Milky Way as its origin. (This is
distinct from `~astropy.coordinates.Galactic` coordinates, which point
toward the Galactic Center but have their origin in the Solar System).
The `~astropy.coordinates.Galactocentric` frame
```

    For an example of how to use the `~astropy.coordinates.Galactocentric`
    frame, see
    :ref:`sphx_glr_generated_examples_coordinates_plot_galactocentric-frame.py`.

.. _astropy-coordinates-galactocentric-defaults:

Controlling the Default Frame Parameters
========================================

All of the frame-defining parameters of the
`~astropy.coordinates.Galactocentric` frame are customizable and can be set by
passing arguments in to the `~astropy.coordinates.Galactocentric` initializer.
However, it is often convenient to use the frame without having to pass in every

> parameter. We have therefore tried to set reasonable default values for these

Reviewer: ```suggestion
parameter. Hence, the class comes with reasonable default values for these
```
